### PR TITLE
Fixes in filter creation and edition dialog

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/components/dialogs/explicit-naming-filter-dialog-content.js [hunk 1]
```diff
@@ -162,6 +153,20 @@ export const ExplicitNamingFilterRow = ({
     );
 };
 
+function generNamingsArray(defaultValues, isGeneratorOrLoad) {
```
Comment (ayolab) [inline src/components/dialogs/explicit-naming-filter-dialog-content.js]: can you change the name of the function to "generateNamingArray"

**Review by ayolab — COMMENTED** [review]
Tests OK

File: src/components/dialogs/explicit-naming-filter-dialog-content.js (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/dialogs/explicit-naming-filter-dialog-content.js]: Done

**Review by ayolab — APPROVED** [review]

**Review by YenguiSeddik — COMMENTED** [review]
Tests:
- We can create explicit naming filter with 0 equipments. i think that should not be possible
- When editing explicit naming filter and we select all the rows and delete them, the "validate" and "cancel" button change their position in the middle of the dialogue.

**Comment by FranckLecuyer** [comment]
> Tests:
> 
>     * We can create explicit naming filter with 0 equipments. i think that should not be possible
> 
>     * When editing explicit naming filter and we select all the rows and delete them, the "validate" and "cancel" button change their position in the middle of the dialogue.

Modifications done.

**Review by YenguiSeddik — APPROVED** [review]
Test: ok
Code review: ok
No warnings in console

**Pull request merged**